src/Nethermind/Nethermind.Init/Modules/WorldStateModule.cs

```diff
@@ -66,6 +67,7 @@ protected override void Load(ContainerBuilder builder)
             .RegisterSingletonJsonRpcModule<IPruningTrieStateAdminRpcModule>()
 
             .AddSingleton<IReadOnlyStateProvider, ChainHeadReadOnlyStateProvider>()
+            .AddSingleton<IVisitingWorldState>(ctx => ctx.Resolve<IWorldStateManager>().GlobalWorldState)
```

Comment:
This might be bad? @asdacap

Reply:
```
Autofac.Core.DependencyResolutionException : None of the constructors found with 'Nethermind.Core.Container.NethermindConstructorFinder' on type 'Nethermind.Core.Test.Modules.MainBlockProcessingContext' can be invoked with the available services and parameters:
Cannot resolve parameter 'Nethermind.State.IVisitingWorldState WorldState'
```
Without this line some tests fail with this exception. I tried to understand how it works with just IWorldState. No success. Maybe you know or @asdacap

Reply:
Not sure if link work. Page hangs. Its in `TestBlockProcessingModule`.